packages/design-system/src/types/ChoiceList/ChoiceList.d.ts

```diff
@@ -1,10 +1,22 @@
 import * as React from 'react';
-import { ChoiceProps } from './Choice';
+import { ChoiceProps as ChoiceComponentProps } from './Choice';
 
 export type ChoiceListSize = 'small';
 
 export type ChoiceListType = 'checkbox' | 'radio';
 
+// Omit props that we override with values from the ChoiceList
+export type ChoiceProps = Omit<
+  ChoiceComponentProps,
+  'inversed',
+  'name',
+  'onBlur',
+  'onChange',
+  'size',
+  'type',
+  'inputRef'
+>;
```

Comment:
```suggestion
// Omit props that we override with values from the ChoiceList
type OmitChoiceProps = 'inversed' | 'name' | 'onBlur' | 'onChange' | 'size' | 'type' | 'inputRef';
export type ChoiceProps = Omit<
  ChoiceComponentProps, OmitChoiceProps
>;
```

Reply:
I also didn't realize that the other type defs were using a different syntax